bottle_server.py

```diff
@@ -58,6 +60,19 @@ def route_static(filepath=None):
         return template('index', root=appPath, get_url=app.get_url)
     return static_file(filepath, root=join(appPath, 'static'))
 
+@get('/version', name='version')
+def get_version():
+    version_backend=subprocess.Popen(['./shell-scripts/run-version-checker.sh', CF], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    (stdout, stderr)=version_backend.communicate()
+
+    if version_backend.returncode != 0:
+        print("Error: CheckerPrinter failed %d %s %s" % (version_backend.returncode,stdout, stderr))
```

Comment:
The `run-version-checker.sh` script doesn't run `CheckerPrinter`. Adapt to something suitable.